src/main/java/sirius/web/services/PublicServices.java

```diff
@@ -65,11 +73,31 @@ public void recordPublicService(Method route) {
             return;
         }
 
+        // collect shared parameters
+        List<Parameter> sharedParameters = new ArrayList<>();
+        Arrays.stream(route.getAnnotationsByType(ParametersFrom.class)).forEach(parametersFrom -> {
+            Collection<SharedParameters> possibleOrigins =
+                    globalContext.getParts(SharedParameters.class).stream().filter(sharedParameter -> {
+                        return parametersFrom.value().isAssignableFrom(sharedParameter.getClass());
+                    }).toList();
```

Comment:
This attempts to simplify the registration of shared parameter instances. We could go for `@Register(classes = MySharedParams.class)` as well.

Reply:
I prefer the @Register

Reply:
There is a @Register – This code removes the need to explicitly add a class there and allows to go with a plain annotation.